muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/tsconfig.json

```diff
@@ -2,18 +2,19 @@
   "compilerOptions": {
     "outDir": "../dist/",
     "allowJs": true,
-    "sourceMap": true,
+    "sourceMap": false,
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "noImplicitAny": true,
     "module": "commonjs",
-    "target": "es5",
+    "target": "esnext",
```

Comment:
Voisi olla parempi käyttää ennemin es6 arvoa jos päädytään nostamaan versiota mihin käännetään fronttikoodit.

[https://www.typescriptlang.org/tsconfig#target](url)

Viittaan lähinnä tähän mainintaan:

_The special ESNext value refers to the highest version your version of TypeScript supports. This setting should be used with caution, since it doesn’t mean the same thing between different TypeScript versions and can make upgrades less predictable._

Reply:
Voisi olla paikallaan käyttää es2017 targettia. Näyttää olevan melko hyvin tuettu: https://caniuse.com/?search=es2017